lib/testmap.py

```diff
@@ -58,6 +58,7 @@
             'fedora-testing/dnf-copr',
             'rhel-9-0-distropkg',
             'fedora-34/firefox-devel',
+            'fedora-35',
```

Comment:
This needs to be separate PR as when you want to trigger `fedora-35` context in this PR, bots already need to know about it.